### DISCUSSION TITLE
ltmeat flood unrealiable

### DISCUSSION THREAD

**Discussion by wvminecraftkid** [3kh0/ext-remover#1166] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I'm using ltmeat flood and it sometimes works but lots of time it only stays for a few minutes before getting blocked again. 

My process

1. Open the window (I'm using 740 tabs because any more and my Chromebook crashes.)
2. Close the window as soon as it opens (is this correct or should I wait for all the tabs to load?)
3. Immediatly open the window again
4. As fast as possible flip allow access to file urls
5. Close the window

This is pretyy unrealiable so I think I'm doing something wrong. What should I improve? Sometimes it ends up working and lasts for around 48 hours but usally not

**Comment by S-PScripts** [comment]
experiment with amount of tabs

or use another exploit
